scripts/legislation/africa/egypt_english.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+function remove_all_text_before_first_header {
+  sed -n '/^FINAL PROVISIONS (ARTICLES 102 to 104).*$/,$p' | \
+    sed -n '/^PRELIMINARY PART/,$p'
+}
+
+function replace_parentheses_around_article_delimiters_with_square_brackets {
+  sed -E 's/^\(([A-Za-z0-9]+)\)/[\1]/'
+}
+
+function remove_all_text_after_last_article {
+  sed -z 's/EXECUTIVE REGULATION OF LAW.*$//' 
+}
+
+function amend_errors_in_headers {
+  sed -E 's/(PART )([A-Z]+)/\n\n\1\2 -/g' | \
+    sed -E 's/(CHAPTER )([A-Z]+)/\n\n\1\2 -/g' | \
+    sed -E 's/(Section )([A-Za-z]+)/\n\n\1\2 -/g' | \
```

Comment:
You should be able to replace these three lines with:

```
sed -E 's/(PART [A-Z]+)/\n\n\1 -/g' | \
    sed -E 's/(CHAPTER [A-Z]+)/\n\n\1 -/g' | \
    sed -E 's/(Section [A-Za-z]+)/\n\n\1 -/g' | \
```

Reply:
Yes, this works.